docs/dsl_inspec.md

```diff
@@ -52,7 +52,7 @@ where
 * `ref` is a reference to an external document
 * `describe` is a block that contains at least one test. A `control` block must contain at least one `describe` block, but may contain as many as required
 * `sshd_config` is an InSpec resource. For the full list of InSpec resources, see InSpec resource documentation
-* `its('Port')` is the matcher; `{ should eq('22') }` is the test. A `describe` block must contain at least one matcher, but may contain as many as required
+* `its('Port')` is the matcher; `{ should eq '22' }` is the test. A `describe` block must contain at least one matcher, but may contain as many as required
```

Comment:
I mean, this is literally wrong, but now it's well-formatted and wrong, so ... :100:

Reply:
Hah! Yeah didn't even read for content. Quite the hilarity though second time through.

Reply:
/me gestures to the can of worms we are avoiding :smile: